aten/src/THC/THCAllocator.cpp

```diff
@@ -4,21 +4,30 @@ THCIpcDeleter::~THCIpcDeleter() {
   int prev_device;
   THCudaCheck(cudaGetDevice(&prev_device));
   THCudaCheck(cudaSetDevice(device_));
-  THCudaCheck(cudaIpcCloseMemHandle(data_));
   THCudaCheck(cudaSetDevice(prev_device));
 }
 
 void deleteTHCIpcDeleter(void* ptr) {
   delete static_cast<THCIpcDeleter*>(ptr);
 }
 
-at::DataPtr THCIpcDeleter::makeDataPtr(void* data, int device) {
+// Refer to NB [CUDA IPC and the caching allocator] for more details
+// basePtr - device ptr of allocated CUDA memory region. This memory region
```

Comment:
nit: "memory region" here is a bit ambiguous. A more specific moniker is "a single cudaMalloc allocation; this may be a large block of memory which is managed by the caching allocator".